packages/web/src/public-site/pages/landing-page/components/FeaturedContent.tsx

```diff
@@ -105,12 +104,15 @@ type FeaturedContentProps = {
 
 const FeaturedContent = (props: FeaturedContentProps) => {
   const { history } = useHistoryContext()
-  const trendingPlaylistsResponse = useAsync(async () => {
-    const featuredContent = await fetchExploreContent(env.EXPLORE_CONTENT_URL)
-    const ids = featuredContent.featuredPlaylists.slice(0, 4)
+  const { value: featuredPlaylists, loading } = useAsync(async () => {
```

Comment:
While we're here, why not make this a tan-query hook

Reply:
it's outside of the app-providers context so have to go manual here sadly

Reply:
ohh public-site. dang